QualifiedTeachersApi/src/QualifiedTeachersApi/DataStore/Crm/DataverseAdapter.GetQtsAwardeesForDateRange.cs

```diff
@@ -0,0 +1,122 @@
+﻿using Microsoft.Xrm.Sdk.Query;
+using QualifiedTeachersApi.DataStore.Crm.Models;
+
+namespace QualifiedTeachersApi.DataStore.Crm;
+
+public partial class DataverseAdapter
+{
+    public async Task<QtsAwardee[]> GetQtsAwardeesForDateRange(DateTime startDate, DateTime endDate)
+    {
+        var filter = new FilterExpression(LogicalOperator.And);
+        filter.AddCondition(dfeta_businesseventaudit.Fields.CreatedOn, ConditionOperator.GreaterEqual, startDate);
+        filter.AddCondition(dfeta_businesseventaudit.Fields.CreatedOn, ConditionOperator.LessEqual, endDate);
```

Comment:
Worth making this operator `LessThan` so that we can do back-to-back queries without any overlap?

Reply:
Done